### PR TITLE
Update hfss.py

### PR DESCRIPTION
**Pull request by PipKat**

### CODE REVIEW

File: pyaedt/hfss.py [hunk 1]
```diff
@@ -102,11 +101,40 @@ def __exit__(self, ex_type, ex_value, ex_traceback):
...
+        LumpedRLC :
+        FiniteCond :
+        """
         (PerfectE, PerfectH, Aperture, Radiation, Impedance, LayeredImp, LumpedRLC, FiniteCond) = range(0, 8)
 
     @aedt_exception_handler
     def _create_boundary(self, name, props, boundary_type):
+        """Create a boundary.
+        
+        Parmeters
```
Comment (PipKat) [inline pyaedt/hfss.py:123]:
```suggestion
        Parameters
```

File: pyaedt/hfss.py [hunk 2]
```diff
@@ -102,11 +101,40 @@ def __exit__(self, ex_type, ex_value, ex_traceback):
...
+        ---------
+        name : st
+            Name ofthe boundary
```
Comment (PipKat) [inline pyaedt/hfss.py:126]:
```suggestion
            Name of the boundary
```

File: pyaedt/hfss.py [hunk 3]
```diff
@@ -484,6 +512,32 @@ def create_linear_step_sweep(self, setupname, unit, freqstart, freqstop, step_si
...
+        current_conformance, str optional
+            The default is ``"Disable'``.
+        thin_sources : bool, optional
+             The default is ``True``.
+        power_fraction str, optional
+             The default is ``"o.95"``.
+        
+        Returns
+        -------
+        type
```
Comment (akaszynski) [inline pyaedt/hfss.py:538]: Return appears to just be the standard success bool.

File: pyaedt/hfss.py [hunk 4]
```diff
@@ -102,11 +101,40 @@ def __exit__(self, ex_type, ex_value, ex_traceback):
...
+        """
         (PerfectE, PerfectH, Aperture, Radiation, Impedance, LayeredImp, LumpedRLC, FiniteCond) = range(0, 8)
 
     @aedt_exception_handler
     def _create_boundary(self, name, props, boundary_type):
+        """Create a boundary.
+        
+        Parameters
+        ---------
+        name : st
```
Comment (akaszynski) [inline pyaedt/hfss.py:125]:
```suggestion
        name : str
```

File: pyaedt/hfss.py [hunk 5]
```diff
@@ -484,6 +512,32 @@ def create_linear_step_sweep(self, setupname, unit, freqstart, freqstop, step_si
...
+        """Create a linked antenna.
+        
+        Parameters
+        -------------
+        source_object :
+        
+        target_cs : str, optional
+            Target coordinate system. The default is ``"Global"``.
+        solution : optional
+            The default is ''None''.
```
Comment (akaszynski) [inline pyaedt/hfss.py:524]:
```suggestion
            The default is ``None``.
```

File: pyaedt/hfss.py [hunk 6]
```diff
@@ -484,6 +512,32 @@ def create_linear_step_sweep(self, setupname, unit, freqstart, freqstop, step_si
...
+        fieldtype : str, optional
+            The default is ``"nearfield"``.
+        use_composite_ports : bool, optional
+            Whether to use composite ports. The default is ``True``.
+        current_conformance, str optional
+            The default is ``"Disable'``.
+        thin_sources : bool, optional
+             The default is ``True``.
+        power_fraction str, optional
+             The default is ``"o.95"``.
```
Comment (akaszynski) [inline pyaedt/hfss.py:534]:
```suggestion
             The default is ``"0.95"``.
```

File: pyaedt/hfss.py (hunk 3 above)
Comment (akaszynski) [inline pyaedt/hfss.py:538]:
```suggestion
        bool
            ``True`` when successful, ``False`` when failed.
```

File: pyaedt/hfss.py [hunk 7]
```diff
@@ -830,7 +882,7 @@ def create_wave_port_between_objects(self, startobj, endobject, axisdir=0, imped
         -------
         BoundaryObject
             Boundary object.
-
+
```
Comment (akaszynski) [inline pyaedt/hfss.py:885]:
```suggestion

```

**Review by akaszynski — APPROVED** [review]
Approved with minor suggestions.

**Pull request merged**